dspace/janitor.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "DSpace",
+  "description": "DSpace is a turnkey institutional repository application.",
+  "icon": "https://janitor.technology/img/dspace.svg",
+  "docker": {
+    "image": "janitortechnology/dspace"
+  },
+  "ports": {
+    "22": {
+      "label": "SSH",
+      "proxy": "none"
+    },
+    "8088": {
+      "label": "VNC",
+      "proxy": "https",
+      "preview": true
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https"
+    }
+  },
+  "scripts": {
+    "Build DSpace": "mvn package && cd dspace/target/dspace-installer && ant fresh_install",
```

Comment:
Nit: Unfortunately, `&&` doesn't work in Cloud9 IDE runners. Please make scripts run one command (with arguments) at a time. E.g. you could have:

```js
{
  "Build DSpace": "mvn package",
  "Install DSpace": {
    "cmd": "ant fresh_install",
    "cwd": "/dspace/target/dspace-installer"
  }
```